doc/locale/ja/LC_MESSAGES/news.po

```diff
@@ -8,7 +8,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: groonga storage engine 1.0.1\n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2022-08-04 11:39+0900\n"
+"POT-Creation-Date: \n"
```

Comment:
Could you remove this header?
Because when this header exists the making document with Sphinx is fail.

Reply:
Sure, I have remove it.

Reply:
I have removed it.